src/pages/billing/List.js

```diff
@@ -249,6 +250,15 @@ export const BillingList = _.flow(
     !isOwner && _.map(project => _.includes('Owner', project.role) ? this.setState({ isOwner: true }) : null, billingProjects)
   }
 
+  onClickCreateProject = async () => {
```

Comment:
naming nit: I would probably name this `onCreateProject`, since that's the essence of what this function does. The fact that it's bound to a click is an implementation detail at the point of usage.

Reply:
I almost used `onCreateProject`, but it's not actually called when the project is created. It's called on the user gesture indicating that they want to create a project which has the effect of showing the modal for creating a project. Though now that I think about it, I could argue that `onClickCreateProject` could also sound like something called when clicking a "Create Project" button in the modal.

How about a different naming approach based on what the function actually does? Something like `openCreateProjectModal` or `showCreateProjectModal`? (Granted it does a little more than that, but that's the desired outcome.)

Reply:
Yes, I think either of those would be good options.